PeraWallet/Classes/Accounts/List/Screens/HomeViewController.swift

```diff
@@ -621,11 +619,6 @@ extension HomeViewController {
             [weak self] in
             guard let self, let rootViewController = UIApplication.shared.rootViewController() else { return }
             analytics.track(.recordHomeScreen(type: .swap))
-            guard configuration.featureFlagService.isEnabled(.swapV2Enabled) else {
-                swapAssetFlowCoordinator.resetDraft()
-                swapAssetFlowCoordinator.launch()
```

Comment:
If `swapV2Enabled` in now enabled by default then this code should't stay in the code? If yes then check your other changes in the PR.